plugins/Annotations/API.php

```diff
@@ -330,4 +343,20 @@ private function checkDateIsValid($date, $canBeNull = false)
 
         Date::factory($date);
     }
+
+    private function filterNote(?string $note): ?string
+    {
+        if (empty($note)) {
+            return $note;
```

Comment:
Does it make sense to explicitly convert null to an empty string here?
```suggestion
            return '';
```

Reply:
No. It needs to stay `null` as it otherwise might change the note to an empty string in the update method.